timescaledb/tutorials/grafana/visualizations/time-series.md

```diff
@@ -0,0 +1,246 @@
+
+## Data for Grafana time-series graphs 
+To plot a time-series graph, Grafana requires you to provide a time column and 
+the respective value column. To plot multiple time-series graphs in a single 
+panel you need to provide multiple value columns.
+
+This is an example of valid time-series data:
+```bash
+Time                | Value_1 | Value_2 |
+--------------------+---------+---------+
+2022-02-08 07:30:01 |      10 |       1 |
+2022-02-08 07:31:01 |      15 |       2 |
+2022-02-08 07:32:01 |      20 |       3 |
+2022-02-08 07:33:01 |      25 |       4 |
+2022-02-08 07:34:01 |      30 |       5 |
+```
+
+This tutorial shows you how to:
+*   [Create a time-series graph with raw data](#create-a-time-series-graph-with-raw-data)
+*   [Create a time-series graph with pre-aggregated data using Continuous Aggregates](#create-a-time-series-graph-from-pre-aggregated-data-using-continuous-aggregates)
+*   [Create multiple time-series graphs in a single panel](#create-multiple-time-series-graphs-in-a-single-panel)
+
+## Prerequisites
+Before you begin, make sure you have:
+* Installed Grafana version&nbsp;8.5 or higher
+* Installed [TimescaleDB][install-timescale]
+* Imported the stock trade data from the [Getting Started Tutorial][gsg-data]
+
+<highlight type="note">
+    If you are new to Grafana, see the
+    [Grafana tutorials](https://docs.timescale.com/timescaledb/latest/tutorials/grafana/)
+    to get familiar with creating your first dashboard and visualizations before
+    you start.
+</highlight>
+
+
+The examples in this section use these variables and Grafana functions:
+* `$symbol`: a variable used to filter results by stock symbols.
+* `$bucket_interval`: the interval size to pass to the `time_bucket`
+  function when aggregating data.
+* `$__timeFrom()::timestamptz` & `$__timeTo()::timestamptz`:
+  Grafana variables. You change the values of these variables by
+  using the dashboard's date chooser when viewing your graph.
+
+
+## Create a time-series graph with raw data
+
+A very common use-case of the time-series graph is displaying stock data. It 
+lends itself well to this use-case as no additional computation is required to 
+visualize the data. A time-series graph makes it easy to see if the value of a
+stock is going up or down. 
+
+<procedure>
+
+### Creating a time-series graph with raw data
+
+1.  Add the `$symbol` variable of type `Text box` to the Grafana dashboard.
+
+1.  In Grafana, create a new panel and add this query:
+
+    ```SQL
+    SELECT time,
+        price
+    FROM stocks_real_time
+    WHERE symbol = '$symbol'
+        AND time >= $__timeFrom()::timestamptz AND time < $__timeTo()::timestamptz
+    ORDER BY time;
+    ```
+
+1.  Enter `AMD` in the `symbol` variable. Adjust the time range of your 
+    dashboard if desired. Keep in mind that we use a raw query, therefore long
+    time ranges could return a significant number of rows and be computationally
+    expensive for Grafana to display.
```

Comment:
```suggestion
    dashboard if desired. This plot uses a raw query, so if you set a large
    time range, it might take a long time for Grafana to get all the rows and
    display the data.
```